cli/src/commands/git/push.rs

```diff
@@ -464,15 +464,23 @@ fn update_change_branches(
     branch_prefix: &str,
 ) -> Result<Vec<String>, CommandError> {
     let mut branch_names = Vec::new();
-    for change_arg in changes {
+    let mut all_commits = Vec::new();
+
+    let workspace_command = tx.base_workspace_helper();
+
+    for change in changes {
+        let revs = workspace_command.resolve_some_revsets_default_single(&[change.clone()])?;
+        all_commits.extend(revs);
```

Comment:
Can use `resolve_some_revsets_default_single()` instead, i think

Reply:
You mean `resolve_some_revsets_default_single(changes)?;`? I think this might be a debugging leftover, I'll try to clean it up.

Reply:
Ah, I figured it out, it was an actual bug, where `args.changes == vec![]` and so this would cause the check in `resolve_some_revsets_default_single` for "Empty revision set" to fail, because the input was obviously empty. I'm not sure if it's better to put the check in `git push` or in `resolve_some_revsets_default_single` though, so I left a note. Let me know what you think.

Reply:
Makes sense to check in `git push`. There are lots of other callers of `resolve_some_revsets_default_single()` and presumably at least some of them want to error out on empty revsets.